### PR TITLE
Use inverted dconf logo

### PR DESCRIPTION
**Pull request by mdparker**
The white dconf logo is intended for a dark gray or black background, but that big dark bar across the D homepage is rather ugly. This PR uses a black logo that works on the default background.

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

File: book/contextMacros.ddoc [hunk 1]
```diff
@@ -1 +1,2 @@
-DVER = 2.097
+COZUM_HTML = frontispiece.cozum.html
+DVER = 2.088
```
Comment (dkorpel) [inline book/contextMacros.ddoc:2]: Unrelated change

Comment (mdparker) [reply book/contextMacros.ddoc]: Fixed. That file should really be added to .gitignore, since it's generated.

Comment (thewilsonator) [reply book/contextMacros.ddoc]: see https://github.com/dlang/dlang.org/pull/3324

**Review by dkorpel — APPROVED** [review]
